appinfo/routes.php

```diff
@@ -22,7 +22,7 @@
 return [
 	'routes' => [
 		['name' => 'site#showDefaultPage', 'url' => '/', 'verb' => 'GET'],
```

Comment:
This route also needs another one with the path and showDefaultPage needs to call createResponse with the path parameter as well